pennylane/templates/subroutines/trotter.py

```diff
@@ -216,8 +220,14 @@ def __init__(  # pylint: disable=too-many-arguments
             "base": hamiltonian,
             "check_hermitian": check_hermitian,
         }
+
         super().__init__(time, wires=hamiltonian.wires, id=id)
 
+    def queue(self, context=qml.QueuingManager):
+        context.remove(self.hyperparameters["base"])
+        context.append(self)
+        return self
```

Comment:
Is this an (independent) bug fix? :thinking:

Reply:
Yep.

Reply:
moving to a separate PR.

Reply:
Moved to #5524